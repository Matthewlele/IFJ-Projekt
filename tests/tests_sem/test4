def foo (s1, s2)
  print s1
  print s2
end

string12 = "Ceska klavesnice:"
string2 = "QWERTZ"

foo string1, string2
