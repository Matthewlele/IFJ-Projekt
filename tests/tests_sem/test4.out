expected SEMANTIC ERR: line 9
